react/OrderBy.js

```diff
@@ -40,64 +39,45 @@ export const SORT_OPTIONS = [
   },
 ]
 
-class OrderBy extends Component {
-  static propTypes = {
-    /** Which sorting option is selected. */
-    orderBy: PropTypes.string,
-    /** Returns the link props. */
-    getLinkProps: PropTypes.func,
-    /** Intl instance. */
-    intl: intlShape,
-    /** Render Runtime context */
-    runtime: PropTypes.shape({
-      navigate: PropTypes.func.isRequired,
-    }).isRequired,
-  }
-
-  get sortingOptions() {
+const OrderBy = ({
+  orderBy,
+  getLinkProps,
+  intl,
+}) => {
+  const sortingOptions = () => {
```

Comment:
what do you think about refactoring this to [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo)?